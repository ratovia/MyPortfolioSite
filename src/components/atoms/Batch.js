import React from "react"
import styled from "styled-components"
import { Typography } from "./Typography"

const BatchArea = styled.div`
  padding: 5px 15px;
  border-radius: 15px;
  background-color: #f2cc69;
  display: inline-block;
`
export const Batch = ({ label }) => {
  return (
    <BatchArea>
      <Typography label={label} color="#424242"></Typography>
    </BatchArea>
  )
}
