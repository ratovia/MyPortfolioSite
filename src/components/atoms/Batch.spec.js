import React from "react"
import renderer from "react-test-renderer"
import { Batch } from "./Batch"

describe("Batch", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Batch label="エンジニア" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
