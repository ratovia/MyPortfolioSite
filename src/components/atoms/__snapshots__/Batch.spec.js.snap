// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Batch renders correctly 1`] = `
<div
  className="sc-eCImvq iiYOXJ"
>
  <p
    className="sc-bdvvaa gceiEI"
    color="#424242"
  >
    エンジニア
  </p>
</div>
`;
